ad2games-theme.gemspec

```diff
@@ -13,6 +13,5 @@ Gem::Specification.new do |s|
   s.files     = `git ls-files`.split("\n")
   s.require_paths = ['lib', 'assets', 'fonts']
 
-  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
How about at least `'rails', '>= 3'`?
For this to work https://github.com/ad2games/ad2games-theme/blob/de08d5bef03e6e42dfbf73e3728c90e93ed741cf/lib/ad2games-theme/engine.rb#L5